app/controllers/sessions_controller.rb

```diff
@@ -28,4 +21,20 @@ def destroy
   def authenticated_password? user
     user&.authenticate params[:session][:password]
   end
+
+  def activate_user user
+    if user.activated?
+      log_in user
+      if params[:session][:remember_me] == Settings.checkbox.checked_value
```

Comment:
Dùng toán tử 3 ngôi đi em

Reply:
e viết như này để sửa lỗi bên rubocop ạ, hay chỉnh lại config đọ dài của dòng trong rubocop ạ :)))